### PR TITLE
Improved Readme

### PR DESCRIPTION
**Pull request by jceelen**
Improved readability of lists by better indenting
Updated the procedure for **1.1 Default Apps Script Developer Console Project** to match with the current interface at Google.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -31,47 +31,43 @@ You can do this one of two ways:
 
 #### 1.1 Default Apps Script Developer Console Project
 
-1. Access the automatically created apps script Developer Console Project by
-  1. `Resources` > `Cloud Platform Project`
-  1. Click on the blue link at the top (`project-name - api-project-##########`) to access the correct Developer Console Project
+1. Go to the [Google Cloud Platform](https://console.cloud.google.com)
```
Comment (lricoy) [inline README.md]: Could you also refer to the other option that is by accesing `Resources` > `Cloud Platform Project` please?

File: README.md [hunk 2]
```diff
@@ -31,47 +31,43 @@ You can do this one of two ways:
...
+   1. Search for `Drive` and select the Google Drive API listing.
+   1. Click `Enable API`
 1. Acquire Google Drive Client Secret Credentials
-  1. In the `Credentials` section, select `Add credentials` and choose `OAuth 2.0 client ID`
-  1. In the menu that appears, choose `Other` for the `Application type`.
-  1. Give it any name you like and click `Create`.
-  1. Finally, download your credentials using the `Download as JSON` button to the right.
-    * Save these credentials to a location of your choosing; `~/Downloads` is fine.
-  1. You may close the Developer Console window.
+   1. If you just enabled the Drive API, hit `Create Credentials`, else, go to `Credentials` in the left menu, select `Create credentials` and choose `OAuth client ID`
```
Comment (lricoy) [inline README.md]: maybe `otherwise` instead of `else`?

**Review by lricoy — CHANGES_REQUESTED** [review]
Really nice! Thanks @jceelen! Could you take at look the Google Cloud's  access step please? The other comment is totally optional :)

File: README.md (hunk 1 above)
Comment (jceelen) [reply README.md]: @lricoy: ure, but can you explain from which point you can access `Resources`? I do not see this in the UI in https://console.developers.google.com 
This is where I got lost in the first place :-)

File: README.md (hunk 2 above)
Comment (jceelen) [reply README.md]: Thanks, that's better!

File: README.md (hunk 1 above)
Comment (jceelen) [reply README.md]: @lricoy could you let me know how to proceed so I can update everything? Thanks! :-)

Comment (jceelen) [reply README.md]: @lricoy nevermind, discovered it myself! I added som more improvements and think it is ready!

**Comment by lricoy** [comment]
Hey @jceelen. Sorry for the delay. Was out of town for the last week. I
will take a look tonight. I am glad you found it :)

Em Qua, 6 de dez de 2017 13:17, Jasper <notifications@github.com> escreveu:

File: README.md (hunk 1 above)
Comment (lricoy) [reply README.md]: Hey @jceelen nice changes but I was referring to just entering `https://script.google.com/` and using the above menu of `Resources` > `Cloud Platform Project`.

File: README.md [hunk 3]
```diff
@@ -23,55 +23,54 @@ Using **gapps**, you can develop your Apps Script locally and push files to the
 ## Quickstart
 
 ### 1. Get Google Drive Credentials
+To use gapps you need access to your files on your Google Drive through the API provided by Google. To make this happen you to create a `Developer Console Project`.
```
Comment (lricoy) [inline README.md]: [...] need to [...]

File: README.md [hunk 4]
```diff
@@ -23,55 +23,54 @@ Using **gapps**, you can develop your Apps Script locally and push files to the
 ## Quickstart
 
 ### 1. Get Google Drive Credentials
+To use gapps you need access to your files on your Google Drive through the API provided by Google. To make this happen you to create a `Developer Console Project`. 
+
+__The only purpose of this project is providing access, this is not the Google Script project you will be working on!__.
```
Comment (lricoy) [inline README.md]: What is the intention here? I think I get it but looks a bit misleading IMHO. What do you mean with "it is not the project you will be working on"?

File: README.md (hunk 1 above)
Comment (lricoy) [reply README.md]: Hey @jceelen nice changes but I was referring to just entering https://script.google.com/ and using the above menu of Resources > Cloud Platform Project.

**Review by lricoy — CHANGES_REQUESTED** [review]

**Comment by lricoy** [comment]
Hey @jceelen. Sorry again for the delay and thanks for the changes. Could you take a last look at the comments? Should not take long and then we could merge it :)

File: README.md (hunk 3 above)
Comment (aurelienshz) [reply README.md]: [...] you have to [...]

Comment (jceelen) [reply README.md]: Fixed!

File: README.md (hunk 4 above)
Comment (jceelen) [reply README.md]: It was not clear to me what the purpose of creating this project was. I deleted it.

**Comment by lricoy** [comment]
Thanks @jceelen 👍

**Pull request merged**